### PR TITLE
add amount_required_matched for trustedform v4 page scan

### PR DESCRIPTION
**Pull request by jgrayson676**
## Description of the change

this adds a new appended field to the v4 integration with a value of 'all', 'some', or 'none' depending on how many required scans were matched, mirroring the behavior of the older TF integrations

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change
- [ ] Technical Debt
- [ ] Documentation

## Related tickets

https://app.shortcut.com/active-prospect/story/65371/trustedform-v4-page-scan-rule-improvements

### Development and Testing

- [ ]  Lint rules pass locally.
- [ ]  The code changed/added as part of this pull request has been covered with tests, or this PR does not alter production code.
- [ ]  All tests related to the changed code pass in development, or tests are not applicable.

### Code Review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [ ]  At least two engineers have been added as "Reviewers" on the pull request.
- [ ]  Changes have been reviewed by at least two other engineers who did not write the code.
- [ ]  This branch has been rebased off master to be current.

### Tracking 
- [ ]  Issue from Shortcut/Jira has a link to this pull request.
- [ ]  This PR has a link to the issue in Shortcut.

### QA
- [ ]  This branch has been deployed to staging and tested.

### CODE REVIEW

**Review by ericyd — DISMISSED** [review]
I didn't test locally (not sure if it's technically possible to test this feature locally, I guess I could use a dev cert) but I appreciate you amending the specs for this 👍🏻

File: README.md [hunk 1]
```diff
@@ -4,3 +4,12 @@ This module is for use on the [LeadConduit](http://activeprospect.com/products/l
 
 
 [![Testing](https://github.com/activeprospect/leadconduit-integration-trustedform/workflows/Node.js%20CI/badge.svg)](https://github.com/activeprospect/leadconduit-integration-trustedform/actions)
+
+## Testing on Development and Staging
```
Comment (ericyd) [inline README.md:8]: 🙏🏻 🙇🏻 🔥 🥇

**Review by ericyd — DISMISSED** [review]
🙌🏻

File: README.md (hunk 1 above)
Comment (henriquelakiap) [reply README.md:8]: Excellent!!

**Review by henriquelakiap — APPROVED** [review]

**Review by ericyd — APPROVED** [review]

**Pull request merged**